src/gui/dialogs/preferences_dialog.cpp

```diff
@@ -95,6 +97,19 @@ struct hotkey_sort_by_desc
 	}
 	const gui2::tpreferences::t_visible_hotkeys* hotkey_commands_;
 };
+
+template<bool reverse>
+struct hotkey_sort_by_command
+{
+	hotkey_sort_by_command(const gui2::tpreferences::t_visible_hotkeys& l) : hotkey_commands_(&l) {}
+	bool operator()(int lhs, int rhs) const
+	{
+		return reverse ? hotkey::get_names((*hotkey_commands_)[lhs]->command) < hotkey::get_names((*hotkey_commands_)[rhs]->command)
```

Comment:
...why on earth did he use conditional operators here... :| I'm not going to complain about your change since you're just copying what was already there, but...

```c++
return (reverse ? std::less<std::string>() : std::greater<std::string>())((*hotkey_commands_)[lhs]->command, (*hotkey_commands_)[rhs]->command);
```

...it'd be great if something like that could be made to work. (I'm pretty sure it won't actually work exactly like that though, since `std::less` and `std::greater` are different types.)